php-classes/Slate/CBL/Tasks/TasksRequestHandler.php

```diff
@@ -49,6 +49,13 @@ protected static function buildBrowseConditions(array $conditions = array(), arr
             if ($archivedConditions != false) {
                 $conditions[] = $archivedConditions;
             }
+
+            $includeUnshared = isset($_REQUEST['include_unshared']) && ($_REQUEST['include_unshared']=='true');
+            $sharedConditions = static::getSharedConditions($includeUnshared);
+
+            if ($sharedConditions != false) {
+                $conditions[] = $sharedConditions;
+            }
```

Comment:
```suggestion
            $includeUnshared = isset($_REQUEST['include_unshared']) && ($_REQUEST['include_unshared']=='true');
            $sharedConditions = static::getSharedConditions($includeUnshared);

            if (!$includeUnshared) {
                $conditions['Status'] = 'shared';
            }
```

Reply:
this has been resolved.  line #58 has also been removed as the $sharedConditions variable is no longer being used.